README.md

```diff
@@ -68,6 +68,7 @@ Please contact sales@rightscale.com to learn more.
 - [AWS Publicly Accessible RDS Instances](./security/aws/rds_publicly_accessible/)
 - [AWS Unencrypted S3 Buckets](./security/aws/unencrypted_s3_buckets/)
 - [Check for publicly accessible Azure SQL Managed Instance](./security/azure/sql_publicly_accessible_managed_instance)
+- [AWS Elastic Load Balancer Encryption Check] (./security/aws/unencrypted_elb_listeners/)
```

Comment:
Create two entries instead of linking to the subdirectory.  Also move the policy directories up one directory, remove unencrypted_elb_listeners.  There is no need for the extra level.